pool/pool.go

```diff
@@ -271,6 +292,31 @@ func (p *Pool) validateTx(ctx context.Context, tx types.Transaction) error {
 	return nil
 }
 
+func (p *Pool) pollMinSuggestedGasPrice(ctx context.Context) error {
+	fromTimestamp := time.Now().UTC().Add(-p.cfg.MinSuggestedGasPriceInterval.Duration)
+	gasPrice, err := p.storage.MinGasPriceSince(ctx, fromTimestamp)
+	if err == nil {
+		if err == state.ErrNotFound {
+			log.Warnf("No suggested min gas price since: %v", fromTimestamp)
+			return err
+		} else if err != nil {
```

Comment:
how can the `error be nil` and then be `NotFound` or `not nil`?

Reply:
omg yep I tried to switch them to see something and forgot to fix it (facepalm)